apiserver/plane/api/urls/issue.py

```
@@ -1,13 +1,14 @@
 from django.urls import path
-
 from plane.api.views import (
     IssueAPIEndpoint,
     LabelAPIEndpoint,
     IssueLinkAPIEndpoint,
     IssueCommentAPIEndpoint,
     IssueActivityAPIEndpoint,
     WorkspaceIssueAPIEndpoint,
-    IssueAttachmentEndpoint,
+    IssueAttachmentV2Endpoint,
+    IssueTypeAPIEndpoint,
```

Ruff (0.8.2)

`plane.api.views.IssueTypeAPIEndpoint` imported but unused

Remove unused import: `plane.api.views.IssueTypeAPIEndpoint`

(F401)